packages/react/src/backlink/Backlink.tsx

```diff
@@ -0,0 +1,47 @@
+import { forwardRef, type Ref } from 'react';
+import { cx } from 'cva';
+import { Link as RACLink, type LinkProps } from 'react-aria-components';
+import { ChevronLeft } from '@obosbbl/grunnmuren-icons-react';
+
+type BacklinkProps = {
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+
+  /** The URL to navigate to when clicking the breadcrumb. */
+  href?: string;
+
+  /** Icon styling */
+  iconStyle?: string;
+} & Omit<LinkProps, 'className' | 'style'>;
```

Comment:
Ser ut som vi mangler en variant for "underlined"? I figma ser det ut som Kristian har tenkt at det skal være mulig å velge med eller uten underline.

Reply:
Vill vi ha det, eller skall vi låta dom överstyra genom classname bara?